### DISCUSSION TITLE
Requesting a x86-64-v3 Windows builds

### DISCUSSION THREAD

**Discussion by syota** [gianni-rosato/svt-av1-psy#65] [Q&A] [answered]
Hello,

I am using AMD Ryzen 3900X with Windows 11.
Therefore, the x86-64-v4 build v2.2.1 did not work.
Is it possible to get the x86-64-v3 Windows builds binaries added as well?
Sorry if this is already being addressed.

Thank you.

**Comment by gianni-rosato** [comment]
Hi, and thanks for the request!

As you may have noticed, we're unfortunately pretty slow with getting Windows builds to be available quickly because none of the core development team uses Windows. I'm thinking about making a Discussion post here as a thread for community-built binaries, many of which would presumably be Windows - would that be something that would interest you?

**Reply by syota** [reply]
I would be very happy if community builds become available.
Thank you for considering this positively.

**Reply by gianni-rosato** [reply, marked as answer]
I just made the thread [here](https://github.com/gianni-rosato/svt-av1-psy/discussions/66) and it should become more active in a few hours. Thanks for your patience!
